packages/flutter/lib/src/services/binding.dart

```diff
@@ -415,6 +416,14 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
   void setSystemUiChangeCallback(SystemUiChangeCallback? callback) {
     _systemUiChangeCallback = callback;
   }
+
+  /// Alert the engine that the binding is registered. This instructs the engine to
+  /// register its top level window handler on Windows. This signals that the app
+  /// is able to process "System.requestAppExit" signals from the engine.
+  @visibleForTesting
+  Future<void> initializationComplete() async {
+    SystemChannels.platform.invokeMethod('System.initializationComplete');
```

Comment:
This should also be documented on `SystemChannels.platform` as one of the methods you can invoke on this.